src/Discord/Parts/Part.php

```diff
@@ -61,7 +61,7 @@ abstract class Part implements ArrayAccess, JsonSerializable
      *
      * @var array The array of attributes that can be mass-assigned.
      */
-    protected $fillable = [];
+    protected static $fillable = [];
```

Comment:
The children class need their `$fillable` changed to static as well otherwise PHP will error saying that you tried to redeclare as non-static. Saying that I think it should be `const` instead of `static` as fillables are immutable.

Reply:
Just leaving my note here (since i posted in discord)

[Permissions.php](https://github.com/discord-php/DiscordPHP/blob/master/src/Discord/Parts/Permissions/Permission.php) Part has a mutable `$fillable`, unless modified, it cant be `const`
https://github.com/discord-php/DiscordPHP/blob/master/src/Discord/Parts/Permissions/Permission.php#L110-L111